core/asciidoc.py

```diff
@@ -2,53 +2,27 @@
 import subprocess
 import tempfile
 
-from .boostrenderer import get_body_from_html
 
-
-def convert_adoc_to_html(file_path, delete_file=True):
+def convert_adoc_to_html(input):
     """
     Converts an AsciiDoc file to HTML.
-    If delete_file is True, the temporary file will be deleted after the
-    conversion is complete.
 
-    Note: This returns the full <html> document, including the <head> and
-    <body> tags.
+    Note: This returns an html fragment, not the full <html> document with the
+    <head> and <body> tags.
 
     The asciidoctor package is a Ruby gem, which is why we're using subprocess
     to run the command.
     https://docs.asciidoctor.org/asciidoctor/latest/
 
-    :param file_path: The path to the AsciiDoc file
-    :param delete_file: Whether or not to delete the temporary file after the
-        conversion is complete
+    :param input: The contents of the AsciiDoc file
     """
     result = subprocess.run(
-        ["asciidoctor", "-o", "-", file_path],
+        ["asciidoctor", "-e", "-o", "-", "-"],
```

Comment:
how do we know we are running the latest asciidoctor?

is this the same thing I get when I invoke asciidoc from the command line?

Reply:
This is running asciidoctor from the command line in the docker image. It uses the version there. I'm not sure what version that is, but it didn't change with this PR.